### PR TITLE
Np 729 update publishing

### PR DESCRIPTION
**Pull request by svenngjeto**
Just ignore warnings from robocomment/Amazon CodeGuru - environment variables get substituted in template by envsubst....

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -8,9 +8,9 @@ Metadata:
     Name: Institutions
     Description: Fetching institution and unit data from the Cristin API
     Author: Unit
-    SemanticVersion: 1.0.0
     SpdxLicenseId: MIT
     LicenseUrl: LICENSE
+    Labels: ['${CODEBUILD_RESOLVED_SOURCE_VERSION}', '${GIT_REPO}']
```
Comment (janbj) [inline template.yaml:13]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Found an embedded parameter outside of an &#34;Fn::Sub&#34; at Metadata/AWS::ServerlessRepo::Application/Labels/1

Comment (janbj) [inline template.yaml:13]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Found an embedded parameter outside of an &#34;Fn::Sub&#34; at Metadata/AWS::ServerlessRepo::Application/Labels/0

**Review by brinxmat — COMMENTED** [review]
This lambda will be removed once we have nva-institution-proxy

File: template.yaml (hunk 1 above)
Comment (axthosarouris) [inline template.yaml:13]: Which intrinsic function is this one? 
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference.html

Comment (svenngjeto) [reply template.yaml:13]: Environment variables set via codebuild, used by envsubst to tag the resulting SAM application

**Pull request closed without merging**